src/main/java/org/datadog/jenkins/plugins/datadog/DatadogBuildListener.java

```diff
@@ -129,6 +126,11 @@ public final void onStarted(final Run run, final TaskListener listener) {
     }
   }
 
+  private final boolean isJobTracked(final String jobName) {
```

Comment:
For consistency, I'd be nice to have a method description here.
Something like:

    /**		
     * Checks if a jobName is blacklisted, or not.		
     *		
     * @param jobName - A String containing the name of some job.
     * @return a boolean to signify if the jobName is or is not blacklisted.
     */